src/Helpers/ValidationHelper.cs

```diff
@@ -87,13 +87,13 @@ public static void ValidateWithdrawalAmount(ValidatorEventArgs obj, Boolean isAm
         decimal minimum = Constants.MINIMUM_WITHDRAWAL_BTC_AMOUNT;
         decimal maximum = Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT;
 
-        if (amount < minimum && !isAmountDisabled)
+        if (amount < minimum && isAmountDisabled)
         {
             obj.Status = ValidationStatus.Error;
             obj.ErrorText = $"Error, the minimum amount to withdraw is at least {minimum:f8} BTC";
         }
 
-        if (amount > maximum && !isAmountDisabled)
+        if (amount > maximum && isAmountDisabled)
```

Comment:
I dont think this is correct. It is only validating when the amount IS disabled. Seems wrong.

Reply:
I think you was looking old PR. Refresh the PR.

Reply:
I think you didn't push the commits, I was reviewing the files for that moment.